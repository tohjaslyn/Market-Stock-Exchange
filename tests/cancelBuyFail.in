cancel buy 0000
exit
