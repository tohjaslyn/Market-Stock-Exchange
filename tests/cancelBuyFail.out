$ No such order in buy book.
$ Have a nice day.
